ES/ESmy016.xml

```diff
@@ -31,325 +31,126 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                   <idno>MY-016</idno>
                </msIdentifier>
                <msContents>
-                  <summary>I) Dǝrsanä sanbata krǝstiyān “Homily on the Sabbath of Christians”
-                     (Homily on the observance of Sunday, ascribed to Jacob of Serug) (fols. 1r-33v,
-                     38r-40v), incomplete and misplaced&#xD; II) Malkǝʾa sanbata krǝstiyān “Image of
-                     the Sabbath of Christians”: 25 strophes (fols. 33v-37v, 41r-41v,
-                     43r-44r).</summary>
-                  <msItem xml:id="ms_i1">
-                     <locus from="fols. 1r" to="33v, 38r-40v">fols. 1r-33v, 38r-40v</locus>
-                     <title type="incomplete">I. Dǝrsāna sanbata krǝstiyān “Homily on the Sabbath of
-                        Christians” (Homily on the observance of Sunday, ascribed to Jaqob of
-                        Serug)</title>
+                <summary/>
+                         <msItem xml:id="ms_i1">
+                          <locus from="1r" to="33v"/>
```

Comment:
lists of loci must be separated by commas, otherwise the visualization will be wrong. TEI is just text, not a table.

(actually according to the strict TEI cataloguing rules the description must be according to the foliation, so these are two msItems, with a relationship reconstructing them, but I also always kept the reconstructed sequences)

Reply:
How do you mean this? <locus from="33v" to="37v"/>, <locus from="41r" to="41v"/>, <locus from="43r" to="44r"/>? First it can be like this, in fact I need to get to an edition of Dersana senbat, to improve the description.

Reply:
I mean
1) if you have e.g. `<locus from="1r" to="33v"/><locus from="40r" to="45v"/>` this will be visualized on the app as ff. 13-33v40r-45v or similar
2) the grouping can stay as far as I am concerned, I was only pointing out that the really correct TEI description follows the actual composition of the physical manuscript
so item on ff. 1-33 (part 1 of a text)
then item on ff. 34-39 (part 1 of text 2)
then item ff. 40-45 (part 2 of text 1)
etc etc 
(but in fact I too group, we all do I guess, as we are interested in the texts more, so it can stay, only introduce the punctuation)

Reply:
Sorry, to introduce punctuation where? Between individual loci? What kind of punctuation, commas?

Reply:
leave it for now, will just be an unseparated list, not a big problem

Reply:
![image](https://user-images.githubusercontent.com/17987728/89632063-ec0f9580-d8a1-11ea-914d-408e483be46e.png)
if this is OK for you then it is OK

Reply:
For me, this is fine.